tests/test_deployment.py

```diff
@@ -19,7 +20,12 @@ def test_e2e_deployment(new_deployment_pg, variables):
     actions performed on each page and the yaml values that direct those
     actions are handled by the UIDeploymentRunner class.
     '''
-    runner = UIDeploymentRunner()
+
+    # First parse the config and instantiate the deployment runner
+    config = DeploymentConfig()
```

Comment:
That all sounds good.   My inexperience with pytest led me down this road.  On it and thanks for the verbal help.